src/index.ts

```diff
@@ -102,70 +91,69 @@ export class MagicConnect extends Connector {
 
       // Set the chainId. If no chainId was passed as a parameter, use the chainId from networkOptions
-      this.chainId =
-        desiredChainIdOrChainParameters?.chainId || networkOptions.chainId
+      this.chainId = desiredChainIdOrChainParameters?.chainId || networkOptions.chainId;
     }
   }
 
   private async checkLoggedInStatus() {
     try {
-      const isLoggedIn = await this.magic?.user.isLoggedIn()
-      return isLoggedIn
+      const isLoggedIn = await this.magic?.user.isLoggedIn();
+      return isLoggedIn;
     } catch (error) {
-      return false
+      return false;
     }
   }
 
-  private async handleActivation(
-    desiredChainIdOrChainParameters?: AddEthereumChainParameter
-  ): Promise<void> {
-    const cancelActivation = this.actions.startActivation()
+  private async handleActivation(desiredChainIdOrChainParameters?: AddEthereumChainParameter): Promise<void> {
+    const cancelActivation = this.actions.startActivation();
 
     try {
       // Initialize the magic instance
-      await this.initializeMagicInstance(desiredChainIdOrChainParameters)
+      await this.initializeMagicInstance(desiredChainIdOrChainParameters);
 
-      await this.magic?.wallet.connectWithUI()
+      await this.magic?.wallet.connectWithUI();
 
-      this.setEventListeners()
+      this.setEventListeners();
 
       // Get the current chainId and account from the provider
       const [chainId, accounts] = await Promise.all([
-        this.provider?.request({ method: "eth_chainId" }) as Promise<string>,
+        this.provider?.request({
+          method: "eth_chainId",
+        }) as Promise<string>,
         this.provider?.request({ method: "eth_accounts" }) as Promise<string[]>,
-      ])
+      ]);
 
       // Update the connector state with the current chainId and account
-      this.actions.update({ chainId: parseChainId(chainId), accounts })
+      this.actions.update({ chainId: parseChainId(chainId), accounts });
     } catch (error) {
-      cancelActivation()
+      cancelActivation();
     }
   }
 
   // "autoconnect"
   public async connectEagerly(): Promise<void> {
-    const isLoggedIn = await this.checkLoggedInStatus()
-    if (!isLoggedIn) return
-    await this.handleActivation()
+    const isLoggedIn = await this.checkLoggedInStatus();
+    if (!isLoggedIn) return;
+    await this.handleActivation();
   }
 
   // "connect"
-  public async activate(
-    desiredChainIdOrChainParameters?: AddEthereumChainParameter
-  ): Promise<void> {
-    await this.handleActivation(desiredChainIdOrChainParameters)
+  public async activate(desiredChainIdOrChainParameters?: AddEthereumChainParameter): Promise<void> {
+    await this.handleActivation(desiredChainIdOrChainParameters);
   }
 
   // "disconnect"
   public async deactivate(): Promise<void> {
-    this.actions.resetState()
-    await this.magic?.wallet.disconnect()
-    this.removeEventListeners()
+    this.actions.resetState();
+    await this.magic?.wallet.disconnect();
```

Comment:
`wallet.disconnect` has been deprecated. We should instead use `magic.user.logout`